gradle-plugin-testing/src/main/java/com/palantir/gradle/plugintesting/PluginTestingPlugin.java

```diff
@@ -67,26 +71,31 @@ private static void doApply(Project project) {
                     SourceSet sourceSet = sourceSetContainer.getByName(SourceSet.TEST_SOURCE_SET_NAME);
                     NamedDomainObjectProvider<Configuration> testRuntimeConfig =
                             project.getConfigurations().named(sourceSet.getRuntimeClasspathConfigurationName());
-                    task.getClasspathConfiguration().set(testRuntimeConfig);
+                    Provider<Iterable<ModuleVersionIdentifier>> testRuntimeDependencies = testRuntimeConfig.map(
+                            config -> config.getIncoming().getResolutionResult().getAllComponents().stream()
+                                    .map(ResolvedComponentResult::getModuleVersion)
+                                    .collect(Collectors.toUnmodifiableSet()));
```

Comment:
So before, this did `config.getResolvedConfiguration().getFirstLevelModuleDependencies()` and now we're getting all transitives by using `getAllComponents` - I think we want to keep the old behaviour, as in this plugin we only care about the dependencies that are explicitly listed like so and not the transitives they bring in:

```
dependencies {
    testRuntimeOnly 'something'
}
```